src/shared/modules/discovery/discoveryDuck.js

```diff
@@ -124,14 +134,12 @@ export const discoveryOnStartupEpic = (some$, store) => {
         return Promise.resolve({ type: 'NOOP' })
       }
       if (action.forceURL) {
-        const { username, password, protocol, host } = getUrlInfo(
-          action.forceURL
-        )
+        const { username, protocol, host } = getUrlInfo(action.forceURL)
         updateDiscoveryState(
           {
             ...action,
             username,
-            password,
+            showDbField: !!action.connectTo,
```

Comment:
Not sure if this boolean is the best way to signal if the field should show up or not, was thinking of just checking for the presence of connectTo, but that is a bit awkward when using the version check bellow since there's no value to prefill.

Reply:
Yeah that's a better name!